go.mod

```diff
@@ -3,22 +3,22 @@ module github.com/stolostron/applifecycle-backend-e2e
 go 1.18
 
 require (
-	github.com/go-cmd/cmd v1.3.0
-	github.com/go-logr/logr v1.2.2
-	github.com/go-logr/zapr v1.2.2
-	github.com/go-logr/zerologr v1.2.1
+	github.com/go-cmd/cmd v1.4.1
+	github.com/go-logr/logr v1.2.3
+	github.com/go-logr/zapr v1.2.3
+	github.com/go-logr/zerologr v1.2.2
 	github.com/onsi/ginkgo v1.16.5
-	github.com/onsi/gomega v1.17.0
+	github.com/onsi/gomega v1.20.0
 	github.com/pkg/errors v0.9.1
-	github.com/rs/zerolog v1.26.1
+	github.com/rs/zerolog v1.27.0
 	go.uber.org/zap v1.21.0
-	k8s.io/api v0.23.3
-	k8s.io/apiextensions-apiserver v0.23.3
-	k8s.io/apimachinery v0.23.3
+	k8s.io/api v0.23.9
```

Comment:
same comment here. can the k8s.io/api be upgraded to v0.24.x?

Reply:
Yes it can be upgraded and has been upgraded now. Thank you for pointing that out.